src/config.h

```diff
@@ -122,7 +119,7 @@ sds RSConfig_GetInfoString(const RSConfig *config);
 #define RS_DEFAULT_CONFIG                                                                       \
   {                                                                                             \
     .concurrentMode = 1, .extLoad = NULL, .enableGC = 1, .minTermPrefix = 2,                    \
-    .maxPrefixExpansions = 200, .queryTimeoutMS = 500, .timeoutPolicy = TimeoutPolicy_Return,   \
+    .maxPrefixExpansions = 200, .queryTimeoutMS = 500, .timeoutPolicy = TimeoutPolicy_Fail,     \
```

Comment:
are we sure we want to change it? people might start getting errors that they did not get before ...

Reply:
@gkorland what do you think? should we change it?